### DISCUSSION TITLE
Need help with "Unable to parse source" error

### DISCUSSION THREAD

**Discussion by nadiagorria** [eclipse/windowbuilder#873] [Q&A] [answered]
![WhatsApp Image 2024-08-25 at 6 43 10 PM](https://github.com/user-attachments/assets/75492897-7fb1-4430-9884-e309e0406c0a)
![WhatsApp Image 2024-08-25 at 6 42 44 PM](https://github.com/user-attachments/assets/5d752739-d07c-49c1-b67e-eaa739841fab)
I have a Java SE 17 project with WindowBuilder installed. I followed all the instructional videos on how to install JCalendar, but when I try to use it in any JFrame, these errors appear. I haven't found anything useful yet. Thanks.

**Comment by ptziegler** [comment, marked as answer]
I assume you're talking about https://github.com/eclipse-windowbuilder/windowbuilder/issues/811 and https://github.com/eclipse-windowbuilder/windowbuilder/issues/698?

Can you check whether you have a `module-info.java` in your project and whether it contains `jcalendar` as required module?
https://github.com/eclipse-windowbuilder/windowbuilder/issues/811#issuecomment-2211023208

**Reply by nadiagorria** [reply]
I hadn't noticed that those posts were about the same issue. I've now correctly added JCalendar as a required module, and the error is fixed. Thank you very much! :)
